manifests/install.pp

```diff
@@ -12,12 +12,31 @@
   $proxy_host = $shibboleth_idp::proxy_host
   $proxy_port = $shibboleth_idp::proxy_port
 
+  if $shibboleth_idp::manage_user {
+    ensure_resource('user', $shibboleth_idp::shib_user, {
+        managehome => true,
+        system     => true,
+        gid        => $shibboleth_idp::shib_group,
+        shell      => '/sbin/nologin',
+    })
+
+    ensure_resource('group', $shibboleth_idp::shib_group, {
+        ensure => present
+    })
+  }
+
```

Comment:
You may consider setting the home dir to one of:
```
/var/lib/${shibboleth_idp::shib_user}
/usr/share/${shibboleth_idp::shib_user}  
```
/var/lib/* has all the standard selinux contexts for ssh keys etc, so that might be my choice if you expect to do anything with ssh

I'd do it to the jetty user also if you choose to do this one...  

System users homes are not usually in /home. That is typically for login users only